doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -37,6 +37,111 @@ It is recommended to follow the “url” link values,
 applicable to retrieve resources. Instead of constructing your own URLs,
 to keep your client decoupled from implementation details.
 
+## JSON entity representation
+
+Many operations use the same JSON entity representation format which is described in the present section. An
+entity is represented by a a JSON object which contain the following properties:
+
+* `id` for the entity ID, represented by a JSON string. If the entity has an `id` attribute itself it will be ignored.
+* `type` for the entity type, represented by a JSON string. If the entity has a `type` attribute itself it will be ignored.
+  If the entity doesn't have a type, then this property is not included.
+* A collection of properties, one per entity attribute. Property values can be single values or they can include,
+  `type` and properties representing metadata as well. Property values which correspond to the primitive JSON datatypes
+  "string", "number" and "boolean" and don't include metadadta are rendered as regular JSON values. Property values
+  which specify a custom type or include metadata are rendered as JSON objects with the following properties:
+  * `value`: for the attribute value, rendered as a regular JSON value depending on the type. Attributes with an
+    user-defined informative types are rendered as JSON strings.
```

Comment:
as this TBD I would drop this sentence in this PR. We should track the issue though